### DISCUSSION TITLE
win+c to open calendar not effective

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#699] [Q&A] [answered]
Doesn't matter if Teams is installed or not,
windows 11 Home 22000.466,
ExplorerPatcher version 22000.434.41.6 (https://github.com/valinet/ExplorerPatcher/tree/22000.434.41.6_a5faef1)
It did not happen on the initial 22000.434.41.6(IIRC.)

Pressing Win+C opens Teams dialog regardless of setting.

Didn't happen until last update-cannot isolate if it's Explorerpatcher or Windows update

**Comment by chip33** [comment, marked as answer]
Hi Bra, 
I'm on the same versions of EP and windows 11 (pro) and Win+C opens calendar okay.
Teams is installed for me.

**Reply by BraINstinct0** [reply]
Strange, I have no other Windows-modifying software or installed anything between the behavior change...Any Ideas on how I can narrow down the cause?

Edit: To be precise, there are *three* 22000.434.41.6 releases.
I'm using [this one](https://github.com/valinet/ExplorerPatcher/tree/22000.434.41.6_a5faef1). If it isn't much of a problem, would you mind trying that out(it's the current latest prerelease)

**Reply by valinet** [reply]
Enable the console, check the output. Maybe it doesn’t have symbol info for that, idk… the way it worked is mostly a hack, I will have to rework it and register/handle the shortcut directly to have it work properly.

**Reply by BraINstinct0** [reply]
Seems like symbols loaded correctly, IDK...
```
funchook create 1
[Symbols] Identified known "StartDocked.dll" with hash 7e652d78661ba62e33d41ad1d3180344.
Loaded symbols
Setup user32 functions done
Setup explorer functions done
Setup uxtheme functions done
Setup twinui.pcshell functions done
Setup stobject functions done
Setup bthprops functions done
Setup pnidui functions done
Setup sndvolsso functions done
Setup shell32 functions done
[IME] Context menu patch status: 1
Setup inputswitch functions done
Installed hooks.
Play startup sound thread...
Signal shell ready...
Started "Signal shell ready" thread.
Open Start on monitor thread
Started "Open Start on current monitor" thread.
Progman: 328246
Progman hook: 2294569
[Positioning] Added settings for monitor 0000000000010001 : 0
>>> Signal shell ready.
Ended "Signal shell ready" thread.
Started "Play startup sound" thread.
Ended "Play startup sound" thread.
0 0
[Updates] Starting daemon.
[Updates] Configured update policy on this system: "Check for updates but let me choose whether to download and install them".
[Updates] Path to module: C:\WINDOWS\dxgi.dll
[Updates] Checking against hash "81c0cf39a52c153ab732746c3eee6d01"
[Updates] Update URL: https://api.github.com/repos/valinet/ExplorerPatcher/releases?per_page=1
[Updates] Prerelease update URL: "https://github.com/valinet/ExplorerPatcher/releases/download/22000.434.41.6_a5faef1/ep_setup.exe"
[Updates] Release notes URL: "https://github.com/valinet/ExplorerPatcher/releases/tag/22000.434.41.6_a5faef1"
[Updates] Hash of remote file is "81c0cf39a52c153ab732746c3eee6d01" (valid).
[Updates] No updates are available.
```

Will try reverting to the previous one, and see if issue persists.

**Reply by BraINstinct0** [reply]
Releases since (and including) https://github.com/valinet/ExplorerPatcher/tree/22000.434.41.5_52cc461 shows same behavior. Would it be worth testing more of them?

**Reply by chip33** [reply]
>I'm using this one. If it isn't much of a problem, would you mind trying that out(it's the current latest prerelease) 

hi im also on same pre release ep as you.

**Reply by BraINstinct0** [reply]
Thanks for the info, then this means something might be interfering with my setup..I don't think I'll be able to figure out the cause for now,unfortunately.

**Reply by chip33** [reply]
yes, I had a weird problem with enable classic search bar recently , turned out I never had the new search menu box working at all because of LUA settings.

(https://github.com/valinet/ExplorerPatcher/discussions/669)

**Reply by valinet** [reply]
At some point (soon, I hope), I will rework this to work better without the need on symbols, I now have a new idea, I just don't have the time right now, but bear with me and I will update this to address it and make it work more reliably and with much less fuss. I never knew this was such a popular functionality, as well :D

**Reply by BraINstinct0** [reply]
Good to know that there's a mitigation! This is a surely a non-critical issue, but such a good QoL feature IMO. Always having a great experience with ExplorerPatcher. Thanks *chip33* and *valinet*

**Reply by valinet** [reply]
I implemented a different way of traping `Win`+`C` in the latest pre-release (22000.434.41.10) which should be much more reliable (and does not require symbol data).

**Reply by BraINstinct0** [reply]
Confirming that with the latest pre-release Win+C is working again!

**Reply by valinet** [reply]
Thanks!
